build/packaging/rpm/glideinwms.spec

```diff
@@ -297,11 +299,17 @@ rm -f $RPM_BUILD_ROOT%{python_sitelib}/glideinwms/creation/templates/factory_ini
 rm -f $RPM_BUILD_ROOT%{python_sitelib}/glideinwms/creation/templates/frontend_initd_startup_template_sl7
 rm -f $RPM_BUILD_ROOT%{python_sitelib}/glideinwms/creation/templates/gwms-factory.service
 rm -f $RPM_BUILD_ROOT%{python_sitelib}/glideinwms/creation/templates/gwms-frontend.service
+rm -f $RPM_BUILD_ROOT%{python_sitelib}/glideinwms/creation/templates/gwms-renew-proxies.cron
```

Comment:
I'm assuming we have to remove this due to a poorly written `setup.py`?

(this RPM .spec file is really quite the monster, but cleaning it is out-of-scope for this PR...)

Reply:
It's even worse than that: https://github.com/holzman/glideinWMS/blob/master/build/packaging/rpm/glideinwms.spec#L268